oasislmf/execution/bash.py

```diff
@@ -2285,7 +2327,13 @@ def genbash(
     # remove the file if it already exists
     if os.path.exists(filename):
         os.remove(filename)
-
-    with bash_wrapper(filename, bash_trace, stderr_guard):
+    
+    with bash_wrapper(  
+        filename, 
+        bash_trace, 
+        stderr_guard,
```

Comment:
Trailing whitespace here and a number of other places in this file. VSCode will let you trim trailing whitespace on save.

```suggestion

    with bash_wrapper(
        filename,
        bash_trace,
        stderr_guard,
```